testing/src/test/java/org/motechproject/nms/testing/it/api/MobileAcademyControllerBundleIT.java

```diff
@@ -158,4 +192,469 @@ public void testSmsStatusInvalidFormat() throws IOException, InterruptedExceptio
         assertTrue(SimpleHttpClient.execHttpRequest(request, HttpStatus.SC_BAD_REQUEST, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
     }
 
+    /**
+     * setup MA course structure from nmsCourse.json file.
+     */
+    private JSONObject setupMaCourse() throws IOException {
+        MaCourse course = new MaCourse();
+        String jsonText = IOUtils
```

Comment:
The proper line width is 120 characters, please make sure to line wrap at 120 characters, not 80.